src/app/Dashboard/Charts/diagnostics/DiagnosticsCard.tsx

```diff
@@ -0,0 +1,200 @@
+  DashboardCardDescriptor,
+} from '@app/Dashboard/types';
+import { ErrorView } from '@app/ErrorView/ErrorView';
+import { FeatureLevel } from '@app/Shared/Services/service.types';
+import { LoadingProps } from '@app/Shared/Components/types';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { useSubscriptions } from '@app/utils/hooks/useSubscriptions';
+import {
+  Bullseye,
+  Button,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Label,
+  EmptyStateHeader,
+  EmptyStateFooter,
+} from '@patternfly/react-core';
+import { DataSourceIcon, SyncAltIcon, TachometerAltIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { DashboardCard } from '../../DashboardCard';
+
+export interface DiagnosticsCardProps extends DashboardCardTypeProps {
+  chartKind: string;
+  duration: number;
+  period: number;
+}
+
+export enum DiagnosticsCardKind {}
+
+export function kindToId(kind: string): number {
+  return DiagnosticsCardKind[kind];
+}
+
+export const DiagnosticsCard: DashboardCardFC<DiagnosticsCardProps> = (props) => {
+  const { t } = useTranslation();
+  const serviceContext = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const [running, setRunning] = React.useState(false);
+  const isError = React.useMemo(() => errorMessage != '', [errorMessage]);
+
+  const handleError = React.useCallback(
+    (error) => {
+      setErrorMessage(error.message);
+    },
+    [setErrorMessage],
+  );
+
+  const handleGC = React.useCallback(() => {
+    addSubscription(
+      serviceContext.api.runGC().subscribe({
+        error: (err) => handleError(err),
+      }),
+    );
+  }, [addSubscription, serviceContext.api, handleError]);
+
+  const GCButton = React.useMemo(() => {
+    return (
+      <Button
+        key={0}
+        aria-label={t('DIAGNOSTICS_GC_BUTTON', { chartKind: props.chartKind })}
+        onClick={handleGC}
+        variant="plain"
+        icon={<SyncAltIcon />}
+        isDisabled={false}
+      />
+    );
+  }, [t, props.chartKind, handleGC]);
+
+  const actions = React.useMemo(() => {
+    const a = props.actions || [];
+    return [GCButton, ...a];
+  }, [props.actions, GCButton]);
+
+  const gcButtonLoadingProps = React.useMemo(
+    () =>
+      ({
+        spinnerAriaValueText: 'Invoke GC',
+        spinnerAriaLabel: 'saving-credentials',
+        isLoading: running,
+      }) as LoadingProps,
+    [running],
+  );
+
+  const header = React.useMemo(() => {
+    return (
+      <CardHeader
+        actions={{
+          actions: <>{actions}</>,
+          hasNoOffset: false,
+          className: undefined,
+        }}
+      >
+        <CardTitle>
+          {t('CHART_CARD.TITLE', { chartKind: props.chartKind, duration: props.duration, period: props.period })}
```

Comment:
Reusing this string is why the card has an odd title like this:

![image](https://github.com/user-attachments/assets/5fe4e138-8411-44c0-861f-2b2e42e46780)

so after splitting this card out into its own subdirectory and its own localization "block" (not sure of the proper terminology there), it should also have its own independent set of localization strings for all these card titles and contents.